Misc/NEWS.d/next/Library/2023-07-22-16-44-58.gh-issue-82500.cQYoPj.rst

```diff
@@ -0,0 +1 @@
+Fix overflow on 32-bit systems with :mod:`asyncio` sendfile implemention.
```

Comment:
Could you use the notation to reference specifically the `sendfile` method (assuming there is such a notation)?

Reply:
adjusted

Reply:
c83dd4b55e3e60e0aa8348eab9a2b3dbb4906eea

Reply:
Thanks (you don't need to link to the commit in the comment -- GitHub shows it to me already :-).